Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -409,17 +414,17 @@ void PixelShaderManager::SetFogParamChanged()
 {
   if (!g_ActiveConfig.bDisableFog)
   {
-    constants.fogf[1][0] = bpmem.fog.GetA();
+    constants.fogf[1][2] = bpmem.fog.GetA();
     constants.fogi[1] = bpmem.fog.b_magnitude;
-    constants.fogf[1][2] = bpmem.fog.GetC();
+    constants.fogf[1][3] = bpmem.fog.GetC();
```

Comment:
A/C going from 0/2 to 2/3 looks a little suspicious to me (although I have to admit I have no idea what this is doing :P). Is this alright, or should this be 1/3 or 2/4 instead?

Reply:
I changed the storage for A/C from .x/.z to .z/.w so to save a uniform, as .x/.y are used for the table coefficients. This way it only uses 5 vec4s instead of 6.